### PR TITLE
Add USAGE.md

### PR DESCRIPTION
**Pull request by cchampet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,16 @@ License
 See [**COPYING.md**](COPYING.md)
 
 
+How to use
+-------
```
Comment (fabiencastan) [inline README.md]: Markdown rule is to be aligned with the title.

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ The project is composed of 4 main parts:
 > - **Sam**, a set of command line tools.  
 
 [![Stories in Ready](https://badge.waffle.io/tuttleofx/tuttleofx.svg?label=1 - Ready)](http://waffle.io/tuttleofx/tuttleofx) <br />
-TuttleOFX is alive! 
+TuttleOFX is alive! <br/>
```
Comment (fabiencastan) [inline README.md]: br is not markdown.
To have a line break without a paragraph, you will need to use two trailing spaces.

File: README.md (hunk 1 above)
Comment (cchampet) [inline README.md]: Done here 43f9fe7

File: README.md (hunk 2 above)
Comment (cchampet) [inline README.md]: Done here 43f9fe7

**Pull request merged**